index.html

```diff
@@ -337,6 +356,17 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                                        <span>Member, Embedded Team</span>
                                     </div>
                                  </div>
+                              </article>
+
```

Comment:
You can edit it from here. No need to send a new PR. Also, if you keep doing commits to your local repo, they will automatically be added to the PR.